### ISSUE TITLE
Clarify diseases vs phenotypes, add to Mondo FAQ

### ISSUE THREAD

**Issue by nlharris** [monarch-initiative/mondo#2101]
See also: #559, #477

There is already some material about this on the HPO homepage:
https://hpo.jax.org/app/help/collaboration (scroll down to “The distinction between diseases and phenotypes”).

Add to Mondo FAQ?

**Comment by nlharris** [comment]
(Ticket added at @cmungall's request)

**Comment by nicolevasilevsky** [comment]
How is this proposed text for the Mondo FAQ page (I adapted this from the HPO page, cc @pnrobinson and @cmungall):

Q: What is the difference between a disease and a phenotype?

A: There is a lot of overlap between diseases and phenotypes. Mondo and Human Phenotype Ontology (HPO) define a disease as an entity that has all four of the following features:

- an etiology (whether identified or as yet unknown)
- a time course
- a set of phenotypic features
- if treatments exist, there is a characteristic response to them

A phenotype (or a phenotypic feature) is a component of a disease, and HPO terms can be used to describe the set of phenotypic features that characterize a disease. There is a grey zone between diseases and phenotypic features. For instance, diabetes mellitus can be conceptualized as a disease, but it is also a feature of other diseases such as Bardet Biedl syndrome. In some cases like this, Mondo may label the disease entity with (disease) after the disease name, to distinguish it from the HPO phenotype term (for example, MONDO_0005466 'hypersomnia (disease)' and HP:0100786 Hypersomnia).

See related ticket: https://github.com/monarch-initiative/mondo/issues/477

(I was thinking we could include the link to this Mondo ticket in the FAQ)

**Comment by nlharris** [comment]
that looks good to me! what will the URL be? i can put that in the HPO paper (unless peter would prefer for us to link to https://hpo.jax.org/app/help/collaboration -- but then you have to look down the page to find that info).

not sure i'd put a link to a ticket in an FAQ. if you want the info in the ticket to persist and be visible to people outside the project, it should become a web page itself, or at the least a gdoc or something, not a ticket.

**Comment by nicolevasilevsky** [comment]
The URL will be:
https://mondo.monarchinitiative.org/pages/faq/

You will have to scroll down the page for this too (unless we add it at the top). I was thinking we should add it after (or maybe before) How is the Human Phenotype Ontology (HPO) related to Mondo?

**Comment by nicolevasilevsky** [comment]
Should I go ahead and add this to the site? Or should I wait for Chris and/or Peter to sign off? (we can always update it later...)

**Comment by nicolevasilevsky** [comment]
This is now live on this page

https://mondo.monarchinitiative.org/pages/faq/
